dhall/tests/format/recordLitComments0A.dhall

```diff
@@ -0,0 +1,6 @@
+{ {- A -}
+  x
+  {- B -}
+  = {- C -}
+    T
```

Comment:
Perhaps this could also test simpler usage such as:
```
{
--| a comment for y
y = 42
}
```

This get currently reformatted as:
```
{ --| a comment for y
  y
  =
  42
}
```

And it looks like this contains too many extra-lines.

Reply:
Thanks, @TristanCacqueray. That's a good test case!

I'm getting

```dhall
{ --| a comment for y
  y
  =
    42
}
```

as the output.

I think it should be

```
```dhall
{ --| a comment for y
  y
  = 42
}
```

or ideally

```dhall
```dhall
{ --| a comment for y
  y = 42
}
```

Reply:
I've made a bit of a mess in the code, but achieved the desired output.